src/overlays/actors/ovl_Bg_Haka_Tubo/z_bg_haka_tubo.c

```diff
@@ -164,7 +164,7 @@ void BgHakaTubo_DropCollectible(BgHakaTubo* this, PlayState* play) {
                 func_80078884(NA_SE_SY_CORRECT_CHIME);
                 // Drop rupees
                 for (i = 0; i < 9; i++) {
-                    collectible = Item_DropCollectible(
+                    collectible = (EnItem00*)Item_DropCollectible(
```

Comment:
I feel like the return can just be kept as `Actor*`
(`EnItem00* collectible` -> `Actor* collectible`)
(and in the other spots)

Reply:
There is of course one case that needs it, but I've done the rest.